### ISSUE TITLE
 Visualise iterations periodically during the runtime

### ISSUE THREAD

**Issue by kickbox** [jkrijthe/Rtsne#20]
It would be great if there could be a plotting option after a set number of iterations during the runtime of the algorithm, This is to see the progress and decide on right number of iterations easily.

Right now when I try to find the sweet spot(# iterations) I have to call Rmtse() multiple times with different parameters. This is time consuming and can be optimised with frequent feedback through plots like this http://projector.tensorflow.org/.

**Comment by jkrijthe** [comment]
I agree this would be a nice feature. I looked into using a callback function earlier but settled for adding the ability to start with a user specified initialization to get a similar effect (with a lot more copying of data into and out of memory). The callback function would be nicer, but requires some more work in the C++ code.

**Comment by wudustan** [comment]
@kickbox the alternative package tsne from [J Donaldson](https://github.com/jdonaldson/rtsne/) has callback support. You can use that to optimise your run before feeding it into Rtsne() although, it is a different implementation.
